### DISCUSSION TITLE
可以加个编译选项，不显示xmake的输出，但是显示编译器的输出吗？

### DISCUSSION THREAD

**Discussion by nullptr-yxw** [xmake-io/xmake#4452] [Ideas]
### 你在什么场景下需要该功能？

![image](https://github.com/xmake-io/xmake/assets/96286337/23271a62-8ff9-4d8c-9597-7e7bcf437fca)

如果用`xmake build -q -w`，那么编译器的输出也会跟着一起被删除，和`xmake build -q`的行为一样了就

![image](https://github.com/xmake-io/xmake/assets/96286337/33447964-3b9c-41af-bd12-9edf452c4a4d)

### 描述可能的解决方案

`xmake build -q -w`的时候正常显示编译器输出，只是取消掉xmake的输出

**Comment by waruqi** [comment]
暂不考虑

[a comment by an automated account is not shown]

**Comment by nullptr-yxw** [comment]
> 暂不考虑

好谢谢了，另外xmake watch能不能加个选项，和正常的watch行为一样，每次第一次就正常执行一次，并且之后每次重新开始之前先把之前的内容清空了，和`clear && xmake build -w && xmake run && xmake watch -q -c "sh -c 'clear && xmake build -w && xmake run'"`的行为类似（但事实上还是有点不太一样，这么写是一开始就clear了，watch是下一波输出出来之后再clear之前的）

[a comment by an automated account is not shown]

**Comment by waruqi** [comment]
不考虑，xmake 不会去处理清屏操作。你自己外面处理

[a comment by an automated account is not shown]
